resources/views/ItemList/Components/Filter/ItemFilterImage.twig

```diff
@@ -7,7 +7,7 @@
                     <img :src="value.imageUrl" @error="handleImageError($event)" :width="value.imageUrl ? '80px' : ''"/>
                     <span v-text="value.name"></span>
                 </label>
-                <div class="filter-badge" v-if="value.count" v-text="value.count"></div>
+                <div class="filter-badge" v-if="!value.selected" v-text="value.count"></div>
```

Comment:
Shouldn't this be `value.count && !value.selected`? Otherwise this still may be shown, even if there is no count which could cause problems.